### PR TITLE
Addresses #3, wrong test causing problems, wrong component was compiled

### PR DESCRIPTION
**Pull request by Marauderer97**
Addresses #3 
Changes: found a wrong test case in navbar, changed the component name appropriately 

Demo Link: Not applicable

Screenshots for the change: 

Not applicable
@nikhilrayaprolu @harshit98 Please review

### CODE REVIEW

**Comment by harshit98** [comment]
@nikhilrayaprolu it is saying expect(app-dropdown) element should be present there in the navbar. We have refactored the component earlier. So, in navbar.html `<app-dropdown>` is present. That is why it didn't fail.

**Comment by Marauderer97** [comment]
@harshit98 My bad, edited description accordingly

File: src/app/navbar/navbar.component.spec.ts [hunk 1]
```diff
@@ -81,7 +81,7 @@ describe('Component: Navbar', () => {
   it('should have an app-auto-complete element', () => {
     let compiled = fixture.debugElement.nativeElement;
```
Comment (harshit98) [inline src/app/navbar/navbar.component.spec.ts]: ok. @Marauderer97 => `it('should have an app-dropdown element')`

Comment (Marauderer97) [inline src/app/navbar/navbar.component.spec.ts]: @harshit98 Please check now

**Pull request merged**